src/vm/eetwain.cpp

```diff
@@ -2915,13 +2919,27 @@ void    TRASH_CALLEE_UNSAVED_REGS(PREGDISPLAY pContext)
 
 bool IsMarkerInstr(BYTE val)
 {
-    SUPPORTS_DAC; 
-#ifdef _DEBUG
-    return (val == X86_INSTR_INT3) || // Debugger might stomp with an int3
-           (val == X86_INSTR_HLT && GCStress<cfg_any>::IsEnabled()); // GcCover might stomp with a Hlt
-#else
+    SUPPORTS_DAC;
+
+    if (val == X86_INSTR_INT3)
```

Comment:
This function used to always return false under `_DEBUG`. Now, it will check for int3. What is the implication? It doesn't seem like that change is related to your fix.

Reply:
I believe this is now correct. It means that we allow for a debugger to have inserted int3 instructions. (i.e to have set break points).

Reply:
Also the method  `IsMarkerInstr(BYTE val)` is only ever called in a DEBUG build 
IN asserts and by  CheckInstr* methods.

I will place the entire method under #ifdef _DEBUG

Reply:
I ended up making this method return false for release.